mlflow/statsmodels.py

```diff
@@ -346,39 +362,6 @@ def overrides(klass, function_name):
         except (IndexError, AttributeError):
             return False
 
-    def apply_gorilla_patch(patch, force_backup=True):
```

Comment:
The logic where a patch is applied to a parent class followed by a child class is already handled by `mlflow.utils.gorilla`: https://github.com/mlflow/mlflow/blob/6dee5cb250460e8dc7accb487e54df8c95921e0e/mlflow/utils/gorilla.py#L322-L328